c++/src/Compression.cc

```diff
@@ -899,6 +907,166 @@ DIAGNOSTIC_POP
     return static_cast<uint64_t>(result);
   }
 
+  /**
+   * Block compression base class
+   */
+  class BlockCompressionStream: public CompressionStreamBase {
```

Comment:
Stream compression is not in this change yet?

Reply:
no. this only includes simple block compression